subsys/net/ip/net_mgmt.c

```diff
@@ -182,7 +182,7 @@ static inline void mgmt_run_slist_callbacks(const struct mgmt_event_entry * cons
 	struct net_mgmt_event_callback *cb, *tmp;
 
 	NET_DBG("Event layer %u code %u cmd %u",
-		NET_MGMT_GET_LAYER(mgmt_event->event),
+		NET_MGMT_GET_LAYER(mgmt_event->event) + 1,
```

Comment:
As I mentioned earlier, adding +1 to the value is not clear at all, please add more info why that is needed.

Reply:
Will add a comment.